README.md

```diff
@@ -1,12 +1,12 @@
 # Zulip overview
 
-Zulip is a powerful, open source group chat application that combines the
-immediacy of real-time chat with the productivity benefits of threaded
-conversations. Zulip is used by open source projects, Fortune 500 companies,
-large standards bodies, and others who need a real-time chat system that
-allows users to easily process hundreds or thousands of messages a day. With
-over 500 contributors merging over 500 commits a month, Zulip is also the
-largest and fastest growing open source group chat project.
+Zulip is a powerful, free and open source group chat application that combines
+the immediacy of real-time chat with the productivity benefits of threaded
+conversations. Zulip is used by free and open source projects, Fortune 500
```

Comment:
The phrase "free and open source projects" doesn't make much sense to me. It also isn't a phrase people generally use -- people say "free software project" but not just "free project", nor "free and open source projects".

Reply:
@gnprice what is your proposed solution on what the text should be?